packages/quantic/force-app/main/default/lwc/quanticBreadcrumbManager/quanticBreadcrumbManager.js

```diff
@@ -49,12 +49,11 @@ export default class QuanticBreadcrumbManager extends LightningElement {
   }
 
   renderedCallback() {
-    initializeWithHeadless(this, this.engineId, this.initialize.bind(this));
+    initializeWithHeadless(this, this.engineId, this.initialize);
   }
 
   /** @param {import("coveo").SearchEngine} engine */
-  @api
-  initialize(engine) {
+  initialize = (engine) => {
```

Comment:
I prefer the way it was written initially because it looked more "standard LWC" to me.

Reply:
Darn 😅 
On a scale of 1 to 10 how strongly do you prefer the other way?

Reply:
:smile: Well, maybe 5 out of 10. But, that's a personal preference. In the end, both are giving the same result and we can switch back later on without breaking the API. So if you really prefer the arrow function, I'm fine with that.